src/components/Table/Table.stories.tsx

```diff
@@ -16,6 +17,17 @@ export default {
   parameters: {
     badges: [BADGE.BETA],
   },
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          backgroundColor: 'white',
```

Comment:
Storybook default background same as the zebra hover color, had to change to white for Table stories to differentiate